translations/monero-core_ro.ts

```diff
@@ -454,17 +454,17 @@
     <message>
         <location filename="../LeftPanel.qml" line="346"/>
         <source>R</source>
-        <translation>R</translation>
+        <translation type="unfinished">R</translation>
     </message>
     <message>
         <location filename="../LeftPanel.qml" line="439"/>
         <source>Prove/check</source>
-        <translation type="unfinished"></translation>
+        <translation>Dovadă/Verificare</translation>
     </message>
     <message>
         <location filename="../LeftPanel.qml" line="440"/>
         <source>K</source>
-        <translation>K</translation>
+        <translation type="unfinished">K</translation>
```

Comment:
Looks "finished" to me, correct?
Please check other letters below: B H D M Y E S I - somehow these all got marked as "unfinished".

Reply:
Other than that, looks great. :-)

Reply:
@mandrill-pie take a look at the discussion here about shortcuts: https://github.com/monero-project/monero-gui/pull/1133#discussion_r170714196